peerscout_api/main.py

```diff
@@ -356,6 +369,28 @@ def get_response_json(
         }
 
 
+def write_peerscout_api_response_to_bq(
+    recommendation_request: dict,
+    recommendation_response: dict
+):
+    PROJECT_NAME = 'elife-data-pipeline'
+    DATASET_NAME = get_deployment_env()
+    recommendation_response_with_provenance = remove_key_with_null_value({
+        **recommendation_response,
+        'provenance': {
+            **recommendation_request,
```

Comment:
maybe it would be good to keep it as a `request` field to make it clear where it is coming from?